addons/mail/static/src/discuss_app/discuss.js

```diff
@@ -108,7 +108,9 @@ export class Discuss extends Component {
 
     get channelAvatar() {
         return this.props.public
-            ? url(`/discuss/channel/${this.thread.id}/avatar_128`)
+            ? url(
+                  `/discuss/channel/${this.thread.id}/avatar_128?unique=${this.thread?.avatarCacheKey}`
```

Comment:
in public page, this will have `https://xxx.odoo.com/discuss/channel/17/avatar_128?unique=undefined`

Reply:
Indeed, but it doesn't have any impact, so I'd say it's fine. This is less LOCs so there's that.

Reply:
url method takes kwargs as s second param, we don't need to hardcode them in url

Reply:
with assignDefined it's possible to remove undefined